config/rbac/aggregated_role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: aggregated-manager-role
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      cluster.x-k8s.io/aggregate-to-manager: "true"
```

Comment:
I know we added this to core cluster-api for allowing providers to add roles to the controller-manager for the purposes of reconciling referenced resources. I'm not sure that we need it in this provider, though.

If there are errors caused by something referencing this, then it might be better to update those references instead of adding this here.

Reply:
If there is a reason that we do need it for this provider, then we likely need to chose a different label name, otherwise we'd also aggregate roles that are intended only to be aggregated for the purpose of core cluster-api and not our provider.

Reply:
Just looking at #1575 which seems to want to consume this PR. Yes, it doesn't seem necessary here. Suggest removing this and the reference thereof in #1575.

Reply:
Thank you all! Removed here: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1575/commits/44b7898a696a172d345d0b67cff50004b0580085